src/components/mainMenuButton.js

```diff
@@ -0,0 +1,11 @@
+const mainMenuButton = (className,innerText,callback) => {
+    const button = document.createElement('button');
```

Comment:
wyjaśnisz proszę, dlaczego funkcja przyjmuje argument callback? czemu to służy? chyba nie rozumiem :(

Reply:
_Umożliwić przekazanie do komponentu funkcji callback dla wybranych opcji._

To jedno z zadań.
Chodzi o to, że stworzyłam komponent menu głównego. Komponent menu głównego zawiera w sobie 3 inne komponenty - buttony (People, Quotes, Deaths). Działa to tak, że z poziomu ChooseGameSection przekazuję do komponentu menu głównego funkcję. Co więcej - chcę by ta funkcja wywołała się inaczej w zależności od tego, który przycisk zostanie wciśnięty, dlatego w MainMenu przekazuję ten callback dalej do jego komponentów (buttonów). Dzięki temu, jak spojrzysz sobie na Button, przekazując mu ten callback jesteśmy w stanie wyłuskać nazwę przycisku który został wybrany.

Trochę to skomplikowane, ale mam nadzieję, że chociaż trochę Cię naprowadziłam :)